pandas/core/sparse/frame.py

```diff
@@ -460,7 +497,35 @@ def set_value(self, index, col, value, takeable=False):
         -------
         frame : DataFrame
         """
-        dense = self.to_dense().set_value(index, col, value, takeable=takeable)
+        warnings.warn("get_value is deprecated and will be removed "
```

Comment:
Should this one be "set_value is deprecated..." instead of "get_value"?

Reply:
thanks...i fixed the whole doc-string duplication mess as well.